src/bootstrap.php

```diff
@@ -236,11 +237,29 @@
     }
 
     $content['items'] = $recommendations
-        ->map(function (Model $model) use ($app) {
-            return json_decode($app['elife.api_sdk.serializer']->serialize($model, 'json', [
+        ->map(function (Model $model) use ($app, $type) {
+            $data = json_decode($app['elife.api_sdk.serializer']->serialize($model, 'json', [
                 'snippet' => true,
                 'type' => true,
             ]), true);
+            if ($type->getParameter('version') > 1 && $model instanceof ArticleVersion) {
```

Comment:
two "versions" in the same line that have different meanings, perhaps hide `($type->getParameter('version') > 1` behind a `$shouldContainAbstract` variable